mods/d2k/maps/harkonnen-04/harkonnen04-AI.lua

```diff
@@ -27,7 +27,7 @@ AtreidesTankType = { "combat_tank_a" }
 
 ActivateAI = function()
 	IdlingUnits[fremen] = { }
-	IdlingUnits[atreides] = Reinforcements.Reinforce(atreides, InitialAtreidesReinforcements[1], AtreidesPaths[2]), Reinforcements.Reinforce(atreides, InitialAtreidesReinforcements[2], AtreidesPaths[3])
+	IdlingUnits[atreides] = Utils.Concat(Reinforcements.Reinforce(atreides, InitialAtreidesReinforcements[1], AtreidesPaths[2]), Reinforcements.Reinforce(atreides, InitialAtreidesReinforcements[2], AtreidesPaths[3]))
```

Comment:
`ConcatArrays`?
`MakeTable`?

Seems odd that you pass arrays and get a table as result?

Reply:
Table and array is the same here, i.e. we pass tables and get a table.